securedrop_client/api_jobs/base.py

```diff
@@ -22,6 +22,12 @@ def __init__(self, message: Optional[str] = None) -> None:
         super().__init__(message)
 
 
+class PauseQueueJob(QObject):
+    def __init__(self):
+        super().__init__()
+        self.order_number = 1
```

Comment:
Ooooh, hmmmmm

I think as the client code base continues to grow, we will have more jobs that are not ApiJobs, but for now I like that it's a quick solution and I'll follow-up with filing an issue.

Reply:
Actually, I think it's cleaner to keep `PauseQueueJob` implemented like:
```py
class PauseQueueJob(QObject):
    def __init__(self):
        super().__init__()
        self.order_number = None
```
And then have the `enqueue` function (1) add it to both of the queues after receiving a network timeout error and (2) emit a signal to the controller so it can update the error status bar:
```py
        if isinstance(job, PauseQueueJob):
            self.main_queue.add_job(priority, job)
            self.download_file_queue.add_job(priority, job)
            self.paused.emit()
```